scalability_and_performance/using-node-tuning-operator.adoc

```diff
@@ -22,3 +22,10 @@ include::modules/custom-tuning-specification.adoc[leveloffset=+1]
 include::modules/custom-tuning-example.adoc[leveloffset=+1]
 
 include::modules/node-tuning-operator-supported-tuned-daemon-plug-ins.adoc[leveloffset=+1]
+
+include::modules/node-tuning-hosted-cluster.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+For more information about hosted control planes, see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.6/html/multicluster_engine/multicluster_engine_overview#hosted-control-planes-intro[Using hosted control plane clusters (Technology Preview)].
```

Comment:
Is this documentation on docs.openshift? If so, we can do an `xref::` to the section.

Reply:
Unfortunately, those docs aren't on docs.openshift. They're the docs for Red Hat Advanced Cluster Management.